### PR TITLE
New CAM billing section

### PR DESCRIPTION
**Pull request by gsanchezu**
@rchakra1, @amwatson1s, @bjswoboda, @cristb   please review.

### CODE REVIEW

File: Cloud Application Manager/Billing/billing-menu.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the latest months will be shown. You can also change the range of months being shown in the bar graph by selecting a different value from the to p-right dropdown on the graph.
+
+![billing-dashboard.png](../../images/cloud-application-manager/billing/billing-dashboard.png)
+
+### Billing Usage History
+When you click on **Usage History** option under the **Billing** left menu item, you will be able to review your detailled billing information.
+
+When you enter this page you can see a list with latest months billing summary, and when you click on a line, details for that month appears on the bottom of the page, showing the line-items you may encounter in your invoice, and additional information for each item.
+
+![billing-usage-history.png](../../images/cloud-application-manager/billing/billing-usage-history.png)
```
Comment (jim-phillips) [inline Cloud Application Manager/Billing/billing-menu.md]: change [billing-usage-history.png] to [Billing Usage History]
And the image file is not found

File '/home/travis/build/CenturyLinkCloud/PublicKB/images/cloud-application-manager/billing/billing-usage-history.png' not found. Is there a file extension? Are the file extension and path correct? (Referenced from '/home/travis/build/CenturyLinkCloud/PublicKB/Cloud Application Manager/Billing/billing-menu.md')

File: Cloud Application Manager/Billing/billing-separate-invoices.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+"author": "Guillermo Sanchez",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+* Billing in a single invoice
+* Setting up billing for multiple invoices
+___
```
Comment (jim-phillips) [inline Cloud Application Manager/Billing/billing-separate-invoices.md]: change to empty newline

File: Cloud Application Manager/Billing/billing-menu.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+**In this article:**
+
+* Billing Dashboard
+* Billing Usage History
+___
+### Billing Dashboard
+When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the latest months will be shown. You can also change the range of months being shown in the bar graph by selecting a different value from the to p-right dropdown on the graph.
+
+![billing-dashboard.png](../../images/cloud-application-manager/billing/billing-dashboard.png)
```
Comment (jim-phillips) [inline Cloud Application Manager/Billing/billing-menu.md]: change [billing-dashboard.png] to [Billing Dashboard]
Also image is not found...the paths are case-sensitive.

File '/home/travis/build/CenturyLinkCloud/PublicKB/images/cloud-application-manager/billing/billing-dashboard.png' not found. Is there a file extension? Are the file extension and path correct? (Referenced from '/home/travis/build/CenturyLinkCloud/PublicKB/Cloud Application Manager/Billing/billing-menu.md')

Either, rename the directory to `billing` or update the path to `Billing`

**Comment by jim-phillips** [comment]
@gsanchezu please address comments.

**Comment by gsanchezu** [comment]
Ok @jim-phillips, thanks for your review. I've pushed a commit with them addressed

**Comment by lyzalou** [comment]
@gsanchezu is this ready to be merged? If so, please confirm and mention @asanokhin  and he'll get it into the next deployment. We have one going out tomorrow 7/17 and another on 7/19.

**Comment by gsanchezu** [comment]
@lyzalou  let's wait to see if @rchakra1, @amwatson1s or @bjswoboda have something to add/comment. Thanks!

**Comment by rchakra1** [comment]
@gsanchezu Everything looks good. Can you please make a small change.
For the item, where you had
" Issue a Support Ticket requesting separate invoices by your defined **Cost Centers**"
can you please change to
" Open a Support Ticket request separate invoices by **Cost Centers** and CenturyLink Billing team will assign a separate Billing Account Number for each Cost Center and makes sure that a separate invoice is generated by CostCenter. "

Other than that, everything looks good. Thank you.

**Comment by gsanchezu** [comment]
Hey @lyzalou, @asanokhin, this is now ready to merge. Thanks!

**Pull request merged**